### PR TITLE
Supports rack v2.2.2.

### PR DESCRIPTION
**Pull request by Y-Asahi-dev**
rack v2.2.2対応をお願い致します。

gem 'rack', '2.2.2'
なアプリにて動作確認致しました。

### CODE REVIEW

File: rack-dev-mark.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ['>= 2.3']
 
-  gem.add_dependency "rack", ['>= 1.1', '< 2.1']
+  gem.add_dependency "rack", ['>= 1.1', '<= 2.2.2']
```
Comment (dtaniwaki) [inline rack-dev-mark.gemspec:21]: I think `< 2.3` is fine because a patch version change doesn't have breaking changes.

**Comment by Y-Asahi-dev** [comment]
Thank you for the review.
Corrected the issues pointed out.
Please review again.

**Comment by dtaniwaki** [comment]
Can you resolve the conflicts?

**Comment by Y-Asahi-dev** [comment]
Resolved the conflict.

**Comment by Y-Asahi-dev** [comment]
Thank you!

**Comment by Y-Asahi-dev** [comment]
When will the latest version be released?

**Comment by y-yagi** [comment]
@dtaniwaki Could you please release a new version that includes this fix?

Recently, rack released security patch, but that security patch only supported rack above v2.1. So without this fix, can't use that release.
Ref: https://github.com/rubysec/ruby-advisory-db/blob/master/gems/rack/CVE-2020-8184.yml

**Comment by dtaniwaki** [comment]
I made a new release.

https://github.com/dtaniwaki/rack-dev-mark/releases/tag/v0.7.9

**Comment by y-yagi** [comment]
Thank you so much!

**Pull request merged**